README.md

```diff
@@ -44,28 +44,26 @@ architecture listed below.
 ### Supported operating system versions
 
 * CentOS 7
-* Debian 7
-* (Debian 8)
-* (Debian 9)
-* FreeBSD 10.3
+* Debian 8
+* Debian 9
+* FreeBSD 10.4
 * FreeBSD 11.1
 * Ubuntu 14.04
 * Ubuntu 16.04
-
-_Currently, Debian 8 and Debian 9 are not supported due to a known bug in Zonemaster::LDNS, see [Zonemaster-LDNS issue 10] and [Zonemaster-LDNS issue 30]. 
-That has to be resolved 
-before we can include support for Debian 8 and Debian, respectively._
+* Ubuntu 18.04
 
 ### Supported database engine versions
 
 Operating System | MySQL | PostgreSQL
 ---------------- | ------| -----------
 CentOS 7         | 5.6   |   9.3
-Debian 7         | 5.5   |  10.1
+Debian 8         | TBD   |   TBD
+Debian 9         | TBD   |   TBD
```

Comment:
Debian 9 mysql-server                  5.5.9999+default

Reply:
@sandoche2k, does Debian 9 come with MySQL version 5.5.9999? Do you know what version of PostgreSQL? And Debian 8?

Reply:
For Debian 8 
Mysql - 5.5.60
PostgreSQL- 9.4.15

For Debian 9
Mysql - mysql  Ver 15.1 Distrib 10.1.26-MariaDB (explanation here : https://mariadb.org/debian-9-released-mariadb-mysql-variant)
sh ```
root@vps323915:~# mysql -p
Enter password: 
Welcome to the MariaDB monitor.  Commands end with ; or \g.
Your MariaDB connection id is 4
Server version: 10.1.26-MariaDB-0+deb9u1 Debian 9.1
```
PostgreSQL- 9.6.7